src/ipc/ipc3/host-page-table.c

```diff
@@ -115,7 +115,11 @@ static int ipc_get_page_descriptors(struct dma *dmac, uint8_t *page_table,
 
 	/* source buffer size is always PAGE_SIZE bytes */
 	/* 20 bits for each page, round up to minimum DMA copy size */
+#if CONFIG_ZEPHYR_NATIVE_DRIVERS
+	ret = dma_get_attribute(dmac->z_dev, DMA_ATTR_COPY_ALIGNMENT, &dma_copy_align);
+#else
```

Comment:
We should be able to remove these ifdefs once we have fully partitioned rtos logic.

Reply:
are we actually doing IPC3 with native Zephyr drivers?

Reply:
yes - NXP use this config.